src/dotnet/APIView/APIViewWeb/Models/ReviewCodeFileModel.cs

```diff
@@ -26,5 +26,11 @@ public string Language
 
         [Obsolete("Back compat don't use directly")]
         public bool RunAnalysis { get; set; }
+
+        /// <summary>
+        /// This property marks revisions that were created
+        /// after the revision number was separated off from the name.
+        /// </summary>
+        public bool RevisionNumberSeparateFromName { get; set; }
```

Comment:
Oh, I haven't noticed this. UGH! 

Not a fan of adding a back compat property that would have to be set forever. Can we do a `.StartsWith("rev")` in `DisplayName`?

Reply:
I did that at first, but started thinking about corner cases that could come up..

Reply:
Corner cases would go away with older reviews not being in use anymore. This property would have to stay forever.

Reply:
`new Regex("rev \d+ -")` should cover most.

Reply:
Yes, I meant corner cases where the review name started with "rev". Agree that using a regex would make this much less likely, at a slight perf cost.